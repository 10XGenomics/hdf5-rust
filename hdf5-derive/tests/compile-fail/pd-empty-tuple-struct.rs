#[macro_use]
extern crate hdf5_derive;

use std::marker::PhantomData;

#[derive(H5Type)]
//~^ ERROR proc-macro derive
//~^^ HELP Cannot derive H5Type for empty tuple structs
struct Foo<T>(PhantomData<T>);
