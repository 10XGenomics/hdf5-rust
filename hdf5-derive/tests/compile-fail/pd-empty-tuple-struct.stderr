error: cannot derive `H5Type` for empty tuple structs
 --> $DIR/pd-empty-tuple-struct.rs:9:8
  |
9 | struct Foo<T>(PhantomData<T>);
  |        ^^^
